cf_xarray/accessor.py

```diff
@@ -3,6 +3,11 @@
 import functools
 import inspect
 import itertools
+
+try:
+    from regex import match
+except ImportError:
+    from re import match
 import re
```

Comment:
```suggestion
    from re import match
```

Reply:
`re` is also used on Line 2545. Would you prefer to treat `re.sub` to the same change as `re.match`? I didn't change it since it wasn't necessary.

Reply:
Oh didn't realize that. Let's keep as-is for now then.